src/gazebo/ros_gz_description/models/master_ASM/urdf/master_ASM.urdf

```diff
@@ -76,23 +76,52 @@
  
   <gazebo reference="camera_link">
       <sensor name="camera" type="camera">
-          <pose> 0 0 0 0 0 0 </pose>
           <!-- <visualize>true</visualize> -->
+          <always_on>true</always_on>
           <update_rate>30</update_rate>
           <camera>
-              <horizontal_fov>1.91986218</horizontal_fov>
+            <horizontal_fov>1.1</horizontal_fov> # TODO: This should be 1.91986218 but it doesn't seem to work with a value that high
               <image>
                   <format>R8G8B8</format>
-                  <width>1280</width>
-                  <height>720</height>
-              </image>
-              <clip>
-                  <near>0.05</near>
-                  <far>8.0</far>
-              </clip>
+              <width>1280</width>
+              <height>720</height>
+            </image>
+            <clip>
+              <near>0.3</near>
+              <far>20.0</far>
+            </clip>
           </camera>
           <topic>camera</topic>
       </sensor>
+
+      <sensor name="depthcamera" type="depth">
+          <!-- <visualize>true</visualize> -->
+          <always_on>true</always_on>
+          <update_rate>30</update_rate>
+          <camera>
+        <horizontal_fov>1.1</horizontal_fov> # TODO: This should be 1.91986218 but it doesn't seem to work with a value that high

```

Comment:
idk what is causing this but we should try to fix it

Reply:
I think this is because when it sees over the arena walls it sees infinite depth or something. @pwmarshal, can you please make the arena walls taller for the UCF field?